auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -584,6 +584,21 @@ public void assignResourceRole(String userCrn, String resourceCrn, String resour
         }
     }
 
+    public void assignResourceOwnerRoleIfEntitled(String userCrn, String resourceCrn, String accountId) {
+        try {
+            if (isAuthorizationEntitlementRegistered(userCrn, accountId)) {
+                assignResourceRole(userCrn, resourceCrn, getBuiltInOwnerResourceRoleCrn(), MDCUtils.getRequestId());
```

Comment:
maybe we should use `INTERNAL_ACTOR_CRN` instead of `userCrn` as actor, because it is possible that user does not have the right to assign a role